pkg/dashboard/templates/cost_settings.gohtml

```diff
@@ -36,12 +36,12 @@
       </div>
       <div class="cost-settings-box__inputs-container">
         <label class="cost-settings-box__cost-per-cpu">
-          <span id="cost-settings-box__cost-per-cpu-label-content">Cost per CPU</span>
-          <input id="cost-settings-box__cost-per-cpu-input" placeholder="${X}/month" required disabled pattern="[0-9]+([\.,][0-9]+)?"/>
+          <span id="cost-settings-box__cost-per-cpu-label-content">Cost per CPU-hour</span>
+          <input id="cost-settings-box__cost-per-cpu-input" type="number" step="0.1" required disabled />
```

Comment:
Let's remove `step` - `0.1` is definitely too high, but I'm not sure if `.01` (one cent) is low enough).

Reply:
Weirdly, this still shows up as `text` in firefox. No idea why